### ISSUE TITLE
The primer theme is not currently supported on GitHub Pages.

### ISSUE THREAD

**Issue by wanghenshui** [pages-themes/primer#59]
https://github.com/wanghenshui/cppweeklynews/runs/3143851548

any one seen this? im confused

**Comment by riccardoferretti** [comment]
I have come across a similar error. I am getting an email when pushing on master because of a page build failure.

> Your SCSS file `assets/css/style.scss` has an error on line 1: File to import not found or unreadable: primer. Load path: /tmp/jekyll-remote-theme-20210724-6-quaj2v/_sass.

**Comment by wanghenshui** [comment]
> I have come across a similar error. I am getting an email when pushing on master because of a page build failure.
> 
> > Your SCSS file `assets/css/style.scss` has an error on line 1: File to import not found or unreadable: primer. Load path: /tmp/jekyll-remote-theme-20210724-6-quaj2v/_sass.

i re-run and it builds success, maybe a github pages' bug

**Comment by tfidfwastaken** [comment]
The workaround suggested for https://github.com/pages-themes/cayman/issues/130 seems to have worked with this theme as well.

tldr, in your scss file change `import "{{ site.theme }}"` to `import "jekyll-theme-primer"`
